classes/ActionScheduler_wpCommentLogger.php

```diff
@@ -49,7 +49,10 @@ public function get_entry( $entry_id ) {
 		if ( empty($comment) || $comment->comment_type != self::TYPE ) {
 			return new ActionScheduler_NullLogEntry();
 		}
-		return new ActionScheduler_LogEntry( $comment->comment_post_ID, $comment->comment_content, $comment->comment_type );
+
+		$date = as_get_datetime_object( $comment->comment_date_gmt );
+		$date->setTimezone( ActionScheduler_TimezoneHelper::get_local_timezone() );
+		return new ActionScheduler_LogEntry( $comment->comment_post_ID, $comment->comment_content, $date );
```

Comment:
`as_get_datetime_object()` accepts a `$timezone` as the 2nd parameter and once it is condensed into the `as_get_datetime_object()` call, the `$date` param is no longer used, so you can include all this code on the one line when constructing the `ActionScheduler_LogEntry` instance.

Reply:
I don't think that's going to work. Please correct me if I'm wrong (that's most likely the case 😆 )
`comment_date_gmt` is *ALWAYS* GMT, so at construction time, the `Datetime` should have the UTC timezone.
After the datetime object is created, then we need to set the site's timezone (right before displaying).

Reply:
@cesar-prospress ah yes, you're right. 👍 

And we don't want to use `comment_date`, because we can't guarantee that the site's timezone hasn't changed since that was set. We definitely need to use `comment_date_gmt`.